API.md

```diff
@@ -106,6 +107,14 @@ for sure amount of conditions can be increased endlessly
 
 check that element does not have attribute `disabled`
 
+#### isElementRendered
+- `page` Puppeteer's <Page> instance
+- `selector` <string> that represent CSS Selector
+- `timeout?` Optional. Default timeout is 10000ms
+- returns: `Promise<boolean>`
+
+check is element visible after timeout
```

Comment:
`check is element visible after timeout`
there is a difference between `visible` and `rendered` 
what exactly do you check?
as I see from code you check visibility
might be better to name that function differently
would leave it to you :D